roles/openshift_logging_eventrouter/templates/eventrouter-template.j2

```diff
@@ -42,7 +42,7 @@ objects:
         component: eventrouter
         logging-infra: eventrouter
         provider: openshift
-      replicas: ${REPLICAS}
+      replicas: {{ int(${REPLICAS}) }}
```

Comment:
Does this work as intended?  I would expect this to be evaluated when the template is processed by ansible to take the int value of the env REPLICAS where the previous version would be evaluated when openshift processes the template.  It doesn't look to be equivalent

Reply:
If we wanted ansible to process this and ensure it was a int we would probably use `{{ REPLICAS | int }}`

I think the fix is actually further down in the `PARAMETERS` section, we are using the OCP template parameter logic for `replicas`

Reply:
correct - this doesn't work

Reply:
The eventrouter thing is an OpenShift template problem.
The value is used here an must be a YAML integer:
```yaml
      replicas: ${REPLICAS}
```
The parameter is defined here, and according to an earlier error I had, must be a string?
```yaml
parameters:
  - name: REPLICAS
    displayName: Replicas
    value: "1"
```
Is there some setting to say "when replacing this parameter in the body of the openshift object to create, convert this string value to a YAML integer before sending to openshift"?

Reply:
the answer is "yes"